Extensions/PrimitiveDrawing/Extension.cpp

```diff
@@ -758,9 +758,9 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
       .AddParameter("expression", _("X drawing position"))
       .AddParameter("expression", _("Y drawing position"));
   obj.AddAction("SetAntialiasing",
-                _("Set antialiasing on the shape drawer"),
-                _("(De)Activate antialiasing on the shape drawer"),
-                _("Set antialiasing of _PARAM0_ to _PARAM1_"),
+                _("Set anti-aliasing on the shape drawer"),
+                _("(De)Activate anti-aliasing on the shape drawer"),
+                _("Set anti-aliasing of _PARAM0_ to _PARAM1_"),
```

Comment:
`Set anti-aliasing of _PARAM0_ to _PARAM1_` -> `Enable anti-aliasing on _PARAM0_: _PARAM1_`